src/Camelotia.Presentation.Avalonia/Views/ProviderView.xaml

```diff
@@ -67,16 +80,17 @@
                   </ListBox.ItemsPanel>
                   <ListBox.ItemTemplate>
                     <DataTemplate>
-                      <StackPanel Orientation="Horizontal">
-                        <Button Content="{Binding Name}"/>
-                        <Menu>
-                          <MenuItem Header=">" Items="{Binding Children}"/>
-                        </Menu>
-                      </StackPanel>
+                      <Button>
+                        <StackPanel Orientation="Horizontal">
+                          <TextBlock Text="{Binding Name}"/>
+                          <Menu>                            
+                            <MenuItem Header=">" Items="{Binding Children}"/>
```

Comment:
Looks good so far! Regarding those breadcrumb arrows, we could use an icon from [Avalonia.IconPacks](https://github.com/ahopper/Avalonia.IconPacks). A lot of high-quality vector icons out there, they also have an application that generates XAML code for a selected icon (a `DrawingPresenter` with a `Geometry`).

Reply:
I included some Drawings for the arrows/chevrons. They now also change depending on wether the Menu is open or not. I added them inline though, would it make more sense to define them in a Style/Resources section?

Reply:
Yeah probably worth moving those drawings to `App.xaml` in the future. Probably worth moving the stuff defined in https://github.com/worldbeater/Camelotia/blob/main/src/Camelotia.Presentation.Avalonia/Views/FileView.xaml as well for consistency (no objections about including this change into the diff)